api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,7 +1,7 @@
 //go:build !ignore_autogenerated
 
 /*
-Copyright 2023 Cisco Systems, Inc. and/or its affiliates
+Copyright 2024 Cisco Systems, Inc. and/or its affiliates
```

Comment:
seems unnecessary

Reply:
it is, but CI validation was failing because of it